lib/Whim/Core.pm

```diff
@@ -0,0 +1,314 @@
+package Whim::Core;
+
+use warnings;
+use strict;
+use v5.24;
+
+sub startup {
+    my $self = shift;
+
+    # Add another namespace to load commands from
+    push @{ $self->commands->namespaces }, 'Whim::Command';
+}
```

Comment:
I think this should probably be removed.

Reply:
Unfortunately, removing this also removes `query` and `send`.

Reply:
The namespace is still added in Whim.pm, though. This is in Whim::Core. Unless I'm misunderstanding...

Reply:
Adam's right, I accidentally had this same invocation in both Whim.pm and Whim/Core.pm. Removing the entire startup function from Core.pm leaves everything working just fine AFAICT.